src/fft.js

```diff
@@ -37,16 +43,35 @@ class FFT {
         }
     }
 
+    /**
+     * Create a complex number array with size `2 * size`
+     *
+     * @returns {Float64Array} - A complex number array with size `2 * size`
+     */
     createComplexArray() {
         return new Float64Array(this._csize);
     }
+
+    /**
+     * Converts a complex number representation stored in a Float64Array to an array of real numbers.
+     * 
+     * @param {Float64Array} complex - The complex number representation to be converted.
+     * @param {Array} [storage] - An optional array to store the result in.
+     * @returns {Array} An array of real numbers representing the input complex number representation.
```

Comment:
```suggestion
     * @param {number[]} [storage] - An optional array to store the result in.
     * @returns {number[]} An array of real numbers representing the input complex number representation.
```

To be a bit more specific